posthog-react-native/src/posthog-rn.ts

```diff
@@ -57,6 +57,14 @@ export type PostHogOptions = PostHogCoreOptions & {
    * Experimental support
    */
   sessionReplayConfig?: PostHogSessionReplayConfig
+
+  /**
+   * If enabled, the session id ($session_id) will be persisted across app restarts.
+   * This is an option for back compatibility, so your current data isn't skewed with the new version of the SDK.
+   * If this is false, the session id will be always reset on app restart.
+   * Defaults to false
+   */
+  enablePersistSessionIdAcrossRestart?: boolean
```

Comment:
Another option is `enableRotateSessionIdAcrossRestart` which is enabled by default, and open to ideas.

Reply:
It should be a safe enough change that opting people into the new (and more desirable behaviour) is acceptable

So I like the people have the option to opt back into the old behaviour but that we force upgrade to the new behaviour 👍